### PR TITLE
Refactor insert_c1c2 to use execute_concurrent

### LINKED ISSUE #488: Stop using insert_c1c2
insert_c1c2 only issues updates, but everywhere we use it, it would be more appropriate to use an insert statement. Additionally, refactoring it away in favor of execute_concurrent_with_args seems to be a universally better decision, especially with regards to runtime.

### PR DESCRIPTION
**Pull request by ptnapoleon**
Fixes #488.

@mambocab and @josh-mckenzie  to review

### CODE REVIEW

File: tools.py [hunk 1]
```diff
@@ -27,9 +28,11 @@ def create_c1c2_table(tester, session, read_repair=None):
     tester.create_cf(session, 'cf', columns={'c1': 'text', 'c2': 'text'}, read_repair=read_repair)
 
 
-def insert_c1c2(session, key, consistency=ConsistencyLevel.QUORUM):
-    query = SimpleStatement('UPDATE cf SET c1=\'value1\', c2=\'value2\' WHERE key=\'k%d\'' % key, consistency_level=consistency)
-    session.execute(query)
+def insert_c1c2(session, min_key, max_key, consistency=ConsistencyLevel.QUORUM):
```
Comment (mambocab) [inline tools.py]: I feel like you could avoid some difficult-to-interpret calls like `insert_c1c2(session, 1000, 1001)` by making this just take an iterable:

```
insert_c1c2(session, 1000, 1001) -> insert_c1c2(session, (1000,))
insert_c1c2(session, 1001, 2001) -> insert_c1c2(session, keys=range(1001, 2001))
```

and maybe take a second `n` kwarg for when you just want to insert _N_ keys:

```
insert_c1c2(session, 0, 1000) -> insert_c1c2(session, n=1000)
```

**Comment by mambocab** [comment]
This is excellent. Other than the parameters nitpick, which I'm not super adamant about, this looks good to me. I'd be happy if this were merged as-is, happier if the API were a little nicer.

**Comment by ptnapoleon** [comment]
After consideration, I'm persuaded. I'll make the change.

**Comment by ptnapoleon** [comment]
@mambocab Bump. I made your changes.

File: repair_test.py [hunk 2]
```diff
@@ -96,14 +96,12 @@ def _simple_repair(self, order_preserving_partitioner=False, sequential=True):
 
         # Insert 1000 keys, kill node 3, insert 1 key, restart node 3, insert 1000 more keys
         debug("Inserting data...")
-        for i in xrange(0, 1000):
-            insert_c1c2(session, i, ConsistencyLevel.ALL)
+        insert_c1c2(session, n=1000, consistency=ConsistencyLevel.ALL)
         node3.flush()
         node3.stop()
-        insert_c1c2(session, 1000, ConsistencyLevel.TWO)
+        insert_c1c2(session, keys=(1000, ), consistency=ConsistencyLevel.TWO)
```
Comment (mambocab) [inline repair_test.py]: Don't know why Travis isn't picking up this PR, but `(1000, )` should be `(1000,)`.

Comment (mambocab) [inline repair_test.py]: I think?

File: repair_test.py [hunk 3]
```diff
@@ -297,15 +295,13 @@ def _setup_multi_dc(self):
 
         # Insert 1000 keys, kill node 3, insert 1 key, restart node 3, insert 1000 more keys
         debug("Inserting data...")
-        for i in xrange(0, 1000):
-            insert_c1c2(session, i, ConsistencyLevel.ALL)
+        insert_c1c2(session, n=1000, consistency=ConsistencyLevel.ALL)
         node2.flush()
         node2.stop()
-        insert_c1c2(session, 1000, ConsistencyLevel.THREE)
+        insert_c1c2(session, keys=(1000, ), consistency=ConsistencyLevel.THREE)
```
Comment (mambocab) [inline repair_test.py]: Same here.

**Comment by ptnapoleon** [comment]
Since flake8 doesn't care either way, I'm preferring it as is.

**Pull request merged**